src/main/java/com/es/phoneshop/model/cart/entity/Cart.java

```diff
@@ -0,0 +1,33 @@
+package com.es.phoneshop.model.cart.entity;
+
+import com.es.phoneshop.model.product.entity.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cart {
+    //LinkedHashMap would be better?
```

Comment:
LinkedHashMap is a specific realization, instead of interface. This is the only drawback i could see